apps/judicial-system/web/src/routes/Prosecutor/CreateDetentionRequest/StepOne/StepOne.tsx

```diff
@@ -119,7 +119,6 @@ export const StepOne: React.FC = () => {
       const caseId = await api.createCase({
         policeCaseNumber: policeCaseNumberRef.current.value,
         accusedNationalId: accusedNationalIdRef.current.value.replace('-', ''),
-        court: workingCase.court,
```

Comment:
Are you loosing an older fix by deleting this?

Reply:
YES! Thank you 🙏